float32 north_m
float32 east_m
float32 down_m
float32 north_m_s
float32 east_m_s
float32 down_m_s